lib/errors/InvalidIdentityAssetLockProofHeightError.js

```diff
@@ -0,0 +1,33 @@
+const ConsensusError = require('./ConsensusError');
+
+class InvalidIdentityAssetLockProofHeightError extends ConsensusError {
+  /**
+   *
+   * @param {number} proofCoreChainLockedHeight
+   * @param {number} currentCoreChainLockedHeight
+   */
+  constructor(proofCoreChainLockedHeight, currentCoreChainLockedHeight) {
+    super('Asset lock proof height is greater than consensus height');
```

Comment:
`Asset Lock proof core height ${proofCoreChainLockedHeight} is greater than current consensus core height #{currentCoreChainLockedHeight}.`